### DISCUSSION TITLE
Use blobless git with Bitbucket Server / DataCenter

### DISCUSSION THREAD

**Discussion by phaldan** [renovatebot/renovate#27849] [Suggest an Idea]
### Tell us more.

Since #11759 a full clone is used for Atlassian Bitbucket due to missing support for blobless git. As mentioned in the issue this should be supported by Bitbucket since 7.7. Only Bitbucket 8 is currently still supported and it should be safe to use blobless git with Bitbucket:
https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html

**Comment by rarkins** [comment]
Bitbucket Server/Data Center?

**Reply by viceice** [reply]
yes. I think we already detect the server version on platform init, so we can pass `fullClone=false` if the version is high enough.

**Comment by rarkins** [comment]
Converted to issue: https://github.com/renovatebot/renovate/issues/27853
